tests/puppeteer/campaigns/functional/BO/catalog/brandsAndSuppliers/suppliers/02_filterQuickEditAndBulkActionsSupplier.js

```diff
@@ -0,0 +1,285 @@
+      await this.pageObjects.suppliersPage.filterTable(
+        'input',
+        'name',
+        firstSupplierData.name,
+      );
+      // Check number od suppliers
+      const numberOfSuppliersAfterFilter = await this.pageObjects.suppliersPage.getNumberFromText(
+        this.pageObjects.suppliersPage.gridHeaderTitle,
+      );
+      await expect(numberOfSuppliersAfterFilter).to.be.at.most(numberOfSuppliers);
+      // check text column of first row after filter
+      const textColumn = await this.pageObjects.suppliersPage.getTextContent(
+        this.pageObjects.suppliersPage.tableColumn
+          .replace('%ROW', 1)
+          .replace('%COLUMN', 'name'),
+      );
+      await expect(textColumn).to.contains(firstSupplierData.name);
+    });
+
+    it('should disable first supplier', async function () {
+      const isActionPerformed = await this.pageObjects.suppliersPage.updateEnabledValue(
+        '1',
+        false,
+      );
+      if (isActionPerformed) {
+        const resultMessage = await this.pageObjects.suppliersPage.getTextContent(
+          this.pageObjects.suppliersPage.alertSuccessBlockParagraph,
+        );
+        await expect(resultMessage).to.contains(this.pageObjects.suppliersPage.successfulUpdateStatusMessage);
+      }
+      const isStatusChanged = await this.pageObjects.suppliersPage.elementVisible(
+        this.pageObjects.suppliersPage.enableColumnNotValidIcon.replace('%ROW', 1),
+        100,
+      );
+      await expect(isStatusChanged).to.be.true;
+    });
+
+    it('should enable first supplier', async function () {
+      const isActionPerformed = await this.pageObjects.suppliersPage.updateEnabledValue(
+        '1',
+        true,
+      );
+      if (isActionPerformed) {
+        const resultMessage = await this.pageObjects.suppliersPage.getTextContent(
+          this.pageObjects.suppliersPage.alertSuccessBlockParagraph,
+        );
+        await expect(resultMessage).to.contains(this.pageObjects.suppliersPage.successfulUpdateStatusMessage);
+      }
+      const isStatusChanged = await this.pageObjects.suppliersPage.elementVisible(
+        this.pageObjects.suppliersPage.enableColumnValidIcon.replace('%ROW', 1),
+        100,
+      );
+      await expect(isStatusChanged).to.be.true;
+    });
+
+    it('should reset filter', async function () {
+      const numberOfSuppliersAfterReset = await this.pageObjects.suppliersPage.resetAndGetNumberOfLines();
+      await expect(numberOfSuppliersAfterReset).to.be.equal(numberOfSuppliers);
+    });
+  });
+  // 4: Enable, disable and delete with bulk actions
+  describe('Enable, disable and delete with bulk actions', async () => {
+    it('should disable with bulk actions', async function () {
+      const disableTextResult = await this.pageObjects.suppliersPage.changeSuppliersEnabledColumnBulkActions(false);
+      await expect(disableTextResult).to.be.equal(this.pageObjects.suppliersPage.successfulUpdateStatusMessage);
+
```

Comment:
Please delete the empty line

Reply:
I wanted to add a comment in that line, nice catch